pyodide_build/pywasmcross.py

```diff
@@ -597,7 +605,7 @@ def handle_command(
         if tmp is None:
             # No source file, it's a query for information about the compiler. Pretend we're
             # gfortran by letting gfortran handle it
-            return subprocess.run(line).returncode
+            return subprocess.run(line, check=True).returncode
```

Comment:
We check the result ourselves
```suggestion
        check=False
```

Reply:
```suggestion
            return subprocess.run(line, check=False).returncode
```